test/integration/build_admission_test.go

```diff
@@ -127,8 +139,43 @@ func TestPolicyBasedRestrictionOfBuildConfigCreateAndInstantiateByStrategy(t *te
 	}
 }
 
-func buildStrategyTypes() []string {
-	return []string{"source", "docker", "custom", "jenkinspipeline"}
```

Comment:
I think this comment referred to the test that I removed, but ended up on the wrong line somehow?